Zend/zend_operators.c

```diff
@@ -2983,15 +2983,19 @@ ZEND_API zend_uchar ZEND_FASTCALL _is_numeric_string_ex(const char *str, size_t
 	}
 
 	if (ptr != str + length) {
-		if (!allow_errors) {
-			return 0;
-		}
-		if (allow_errors == -1) {
-			zend_error(E_NOTICE, "A non well formed numeric value encountered");
-			if (EG(exception)) {
-				return 0;
-			}
```

Comment:
this block has been lost in the rebase (easier to see at https://github.com/php/php-src/pull/5762/files?w=1)

Reply:
Indeed, thanks for spotting this :)